lib/bigquery/dataset.js

```diff
@@ -46,20 +53,121 @@ var streamRouter = require('../common/stream-router.js');
  *
  * @alias module:bigquery/dataset
  * @constructor
+ *
+ * @example
+ * var gcloud = require('gcloud');
+ *
+ * var bigquery = gcloud.bigquery({
+ *   keyFilename: '/path/to/keyfile.json',
+ *   projectId: 'grape-spaceship-123'
+ * });
+ * var dataset = bigquery.dataset('institutions');
  */
 function Dataset(bigQuery, id) {
+  var methods = {
+    /**
+     * Create a dataset.
+     *
+     * @example
+     * dataset.create(function(err, dataset, apiResponse) {
```

Comment:
This method is exposed in the docs, but deliberately undocumented in this case, as `create` takes no arguments and having duplicating the callback argument docs is not ideal.

This is a very unique case, though, as most create methods will accept an `options` or `config` object, and we will show param docs that link to the original method (`parentClass.create____`) so the user can see the full list of docs.